indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -325,7 +327,13 @@ public void stop()
     return ImmutableList.of();
   }
 
-  public Collection<ZkWorker> getWorkers()
+  @Override
+  public Collection<Worker> getWorkers()
```

Comment:
@drcrallen this change made the overlord `/druid/indexer/v1/workers` API no longer return a `runningTasks` key for each worker, which may confuse some folks' rolling update scripts and definitely makes the web UI less useful.

how do you feel about having the OverlordResource specifically call `getZkWorkers` if the runner is an RTR? or would it make more sense to add a list of running tasks to the Worker object?